src/constants/core/constants.ts

```diff
@@ -35,7 +35,7 @@ export const TileOutputFormat = {
 } as const;
 
 export type TileOutputFormat = (typeof TileOutputFormat)[keyof typeof TileOutputFormat];
-
+/*  */
 export const TileOutputFormatList = Object.values(TileOutputFormat);
 
 /* eslint-disable @typescript-eslint/no-magic-numbers */
```

Comment:
lets handle error messages inside `CORE_VALIDATIONS`. we should separate the error message from the description

Reply:
i removed description wherever it was useless, added relevant only descriptions for regex constants